project/ProjectPlugin.scala

```diff
@@ -65,97 +61,53 @@ object ProjectPlugin extends AutoPlugin {
     )
 
     lazy val commonTutSettings: Seq[Def.Setting[_]] = Seq(
-      scalacOptions in Tut ~= (_.filterNot(Set("-Ywarn-unused-import", "-Ywarn-dead-code"))),
-      scalacOptions in Tut ++= (scalaBinaryVersion.value match {
-        case "2.10" => Seq("-Xdivergence211")
-        case _      => Nil
-      })
+      scalacOptions in Tut ~= (_.filterNot(Set("-Ywarn-unused-import", "-Ywarn-dead-code")))
     )
 
     lazy val docsSettings: Seq[Def.Setting[_]] =
       micrositeSettings ++ commonTutSettings ++ Seq(aggregate in doc := true)
 
-    lazy val readmeSettings: Seq[Def.Setting[_]] = commonTutSettings ++ Seq(
-      tutSourceDirectory := (baseDirectory in LocalRootProject).value / "tut",
-      tutTargetDirectory := baseDirectory.value.getParentFile,
-      tutNameFilter := """README.md""".r
-    )
-
     lazy val examplesSettings = Seq(
       libraryDependencies ++= Seq(
-        "io.circe"      %% "circe-generic"       % "0.12.0-RC2",
-        "org.tpolecat"  %% s"doobie-core"        % "0.8.0-RC1",
-        "org.tpolecat"  %% s"doobie-h2"          % "0.8.0-RC1",
-        "org.tpolecat"  %% "atto-core"           % "0.7.0",
-        "org.http4s"    %% "http4s-blaze-client" % "0.21.0-M4",
-        "org.http4s"    %% "http4s-circe"        % "0.21.0-M4",
+        "io.circe"      %% "circe-generic"       % "0.13.0",
+        "org.tpolecat"  %% s"doobie-core"        % "0.9.0",
+        "org.tpolecat"  %% s"doobie-h2"          % "0.9.0",
```

Comment:
why the string interpolation? :thinking:

Reply:
It was there before 🤷‍♂️ 

Removed in https://github.com/47degrees/fetch/pull/284/commits/6341994f0649b89f999ffdf9c935b48dbd4fccf0 👍